src/Randomizer.SMZ3/Regions/SuperMetroid/Brinstar/RedBrinstar.cs

```diff
@@ -6,6 +6,8 @@ public class RedBrinstar : SMRegion
     {
         public RedBrinstar(World world, Config config) : base(world, config)
         {
+            // TODO: some of these might expect you to have wall jump, but I'm not sure which or how
```

Comment:
Please check these. The document describes some wall jumps to get through the main red Brinstar shaft but the logic doesn't really take into account how to get there, so I had a tough time with these.

Reply:
I'll have to take a deeper look, but I think the only location that would be accessible via the red Brinstar shaft is AlphaPowerBombRoom, and that currently requires ice beam according to the logic. So currently I don't think you're fully expected to wall jump here.

I do think getting out the top to head back to the ship can require some wall jumps if you don't have the hi jump boots, but you can always go out the left side to avoid that, it just might take longer depending on where you want to go.

Reply:
I guess this is fine as-is, then?